presentation/src/main/java/com/dcs/presentation/trend/TrendRoute.kt

```diff
@@ -0,0 +1,213 @@
+import androidx.compose.foundation.verticalScroll
+import androidx.compose.material3.HorizontalDivider
+import androidx.compose.material3.Scaffold
+import androidx.compose.material3.ScrollableTabRow
+import androidx.compose.material3.Tab
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableIntStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.runtime.setValue
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.res.dimensionResource
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import androidx.hilt.navigation.compose.hiltViewModel
+import androidx.lifecycle.compose.collectAsStateWithLifecycle
+import androidx.paging.PagingData
+import androidx.paging.compose.collectAsLazyPagingItems
+import com.dcs.presentation.R
+import com.dcs.presentation.common.ErrorScreen
+import com.dcs.presentation.common.LoadingScreen
+import com.dcs.presentation.core.model.MovieItemUiState
+import com.dcs.presentation.core.state.MoviePopularType
+import com.dcs.presentation.core.state.MovieTrendType
+import com.dcs.presentation.core.state.UiState
+import kotlinx.coroutines.flow.StateFlow
+
+@Composable
+fun TrendRoute(
+    modifier: Modifier = Modifier,
+    viewModel: TrendViewModel = hiltViewModel()
+) {
+
+    Scaffold { innerPadding ->
+        val scrollState = rememberScrollState()
+
+        Column(
+            modifier = modifier
+                .fillMaxSize()
+                .padding(innerPadding)
+                .verticalScroll(scrollState)
+        ) {
+            TrendMoviePanel(
+                title = stringResource(id = R.string.movie_trend_title_trend),
+                tabs = MovieTrendType.entries.map { it.toUiString() },
+                movies = viewModel.moviesByTrending
+            ) { tabIndex ->
+                viewModel.updateMovieTrendType(MovieTrendType.entries[tabIndex])
+            }
+
+            TrendMoviePanel(
+                title = stringResource(id = R.string.movie_trend_title_popular),
+                tabs = MoviePopularType.entries.map { it.toUiString() },
+                movies = viewModel.moviesByPopular
+            ) { tabIndex ->
+                viewModel.updateMoviePopularType(MoviePopularType.entries[tabIndex])
+            }
+
+            TrendMoviePanel(
+                title = stringResource(id = R.string.movie_trend_title_upcoming),
+                movies = viewModel.moviesByUpcoming
+            )
+        }
+    }
+}
+
+@Composable
+fun TrendMoviePanel(
+    modifier: Modifier = Modifier,
+    title: String,
+    tabs: List<String> = listOf(),
+    movies: StateFlow<UiState<StateFlow<PagingData<MovieItemUiState>>>>,
+    onTabClick: (Int) -> Unit = {}
```

Comment:
```suggestion
    title: String,
    movies: StateFlow<UiState<StateFlow<PagingData<MovieItemUiState>>>>,
    modifier: Modifier = Modifier,
    tabs: List<String> = listOf(),
    onTabClick: (Int) -> Unit = {}
```

컴포저블 함수 param 선언 컨벤션은 기본 param 선언 후에 default 값을 가진 param을 선언하고 그 중 modifier를 최상위에 올리는 것으로 알고 있어요

Reply:
적용 완료